Gemfile

```diff
@@ -10,9 +10,10 @@ gem 'dotenv', require: 'dotenv/load' # Load ENV vars before other gems
 gem 'rails'
 
 # Active Storage
-gem "active_storage_validations"
+gem "active_storage_validations", '~> 3.0.2'
```

Comment:
No need to specify a version here unless there is a good reason to do so. We want dependabot to keep opening PR to upgrade the gem.